### PR TITLE
Add A11y tests for Chips

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add A11y tests for Chip components.

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ToggleChipA11yTest_withSecondaryLabelAndIcon.png
Comment (luizgrp) [inline compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ToggleChipA11yTest_withSecondaryLabelAndIcon.png:1]: This is probably missing a semantics for the labels.

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_SplitToggleChipA11yTest_withSecondaryLabel.png
Comment (luizgrp) [inline compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_SplitToggleChipA11yTest_withSecondaryLabel.png:1]: Will double check if this component should have two separate roles, and if so, if both roles should be called "Checkbox".

**Review by yschimke — APPROVED** [review]

**Comment by kul3r4** [comment]
Thank You!
________________________________

File: compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ToggleChipA11yTest_withSecondaryLabelAndIcon.png
Comment (luizgrp) [reply compose-material/src/test/snapshots/images/com.google.android.horologist.compose.material_ToggleChipA11yTest_withSecondaryLabelAndIcon.png:1]: https://github.com/google/horologist/pull/1399

**Pull request merged**